Cargo.toml

```diff
@@ -10,9 +10,11 @@ repository = "https://github.com/meilisearch/meilisearch-sdk"
 
 [dependencies]
 async-trait = "0.1.51"
-serde_json = "1.0"
+iso8601-duration = "0.1.0"
```

Comment:
Sorry for the newbie here 😅 

I think I get the idea, but just to be sure, the users of the rust-sdk don't need to have these time-handling libraries on their apps? 
In other words, to be able to use some function they need to have the lib, in order to have the right types to pass to the function?

Reply:
The `iso8601-duration` crate is never exposed to the end-user, but sadly the `time` crate will be needed if someone wants to manipulate API keys.

For the context, there was a lot of `TODO: use chrono` (which is one of the biggest crates in the rust ecosystem).
And there was this comment asking for a real « time » type instead of `String` in the SDK: https://github.com/meilisearch/integration-guides/issues/121#issuecomment-944379531

And the reason why we’re not using `chrono` is here: https://github.com/meilisearch/meilisearch-rust/issues/226
There is a bunch (2 or 3, I think) of CVE on `chrono` that are not fixed because the crate is not really maintained anymore.
But for our use case, `chrono` was just a wrapper around `time` (which is maintained and fixed all of the CVE encountered in `chrono`), so I decided to import this one instead of `chrono`.
Also, `time` is way smaller than `chrono`, and you can configure a lot of features depending on your needs.


But yes, to conclude, a user of the `meilisearch-rust` sdk can do most of the thing without any problem, but if he wants to use inspect, create or update anything related to the `OffsetDateTime`, he’ll need to import `time` in his crate.